portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,27 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function getData() {
+  fetch('/data').then(response=>{ console.log(response); return response.json()}).then((data) => {
+    console.log(data);
+    let txt="";
+    for(let i=0;i<data.quotes.length;i++)
+        txt+=data.quotes[i]+"\n";
+    //txt+="你好";
+    //console.log("text is:"+txt);
+    document.getElementById('quote-container').innerText = txt;
+    document.getElementById('pics').innerHTML='\
```

Comment:
The "innerText" of element "quote-container" is not static, the contents of it( comments from visitors) is read from a data storage every time when the page is loaded and will change if new comments get added into the data storage.
The "innerHTML" of element "pics" is static. However, if I write it in index.html, the pictures will be loaded before the comments. As I added a lots of pictures on my page, it will take a very long time before the comments appear on the page. So I write it manually in js to force the comments loaded before all my pictures.

Reply:
@yuxifeng-sps

Reply:
I see, thanks for the explanation!
The code is good as it is for this personal project. Though if it's a group project, it would be helpful to add comments here to explain the consideration behind putting it here.